components/EventsList.js

```diff
@@ -0,0 +1,51 @@
+import Link from "next/link";
+import styled from "styled-components";
+import Head from "next/head";
+import { StyledWrapper } from "@/components/styles/Wrapper";
+import EventCard from "@/components/EventCard";
+
+function EventsList({ entryData }) {
+  return (
+    <StyledWrapper>
+      <Head>
+        <title>Events List</title>
+        <meta name="viewport" content="initial-scale=1.0, width=device-width" />
+      </Head>
+      <Link href="/createEventPage">
+        <p>👉New Event👈</p>
+      </Link>
+      <span>
+        {entryData?.length > 0 ? (
+          <>
+            <h2>⬇️Vote⬇️</h2>
+            <StyledList>
+              {entryData?.map((entry) => (
```

Comment:
you don't need to make this line of code conditional, since you already have the same condition in line 18

```suggestion
              {entryData.map((entry) => (
```